pkg/storage/emptydir/emptydir.go

```diff
@@ -0,0 +1,81 @@
+package emptydir
+
+import (
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+
+	opapi "github.com/openshift/cluster-image-registry-operator/pkg/apis/imageregistry/v1alpha1"
```

Comment:
We should have a follow up PR at some point to unify the nickname for this package across the repository, I think we are currently using at least three different nicknames.  Maybe after @legionus refactoring away from the sdk lands.

Reply:
i copied this one from filesystem.go, so it's not introducing a new nickname anyway.